tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -80,7 +80,6 @@ then
   for RUBY_VERSION in 2.5.0 2.7.0 3.0.0; do
     echo "Installing ruby-${RUBY_VERSION}"
     time rvm install "ruby-${RUBY_VERSION}"
-    time gem install bundler -v 1.17.3 --no-document
     time gem install rake-compiler --no-document
```

Comment:
just as an idea, shouldn't we try to pin the rake compiler version here?

Reply:
I'm wondering if we can actually just delete this `gem install rake-compiler` command, since we should be using the rake-compiler version determined by bundler.

I prefer to save that for another change, though.